test/keys.js

```diff
@@ -10,7 +10,16 @@ describe('keys', function() {
   var cobj = new C();
 
   it("returns an array of the given object's own keys", function() {
-    eq(R.keys(obj).sort(), ['a', 'b', 'c', 'd', 'e', 'f']);
+    eq(R.keys(obj), ['a', 'b', 'c', 'd', 'e', 'f']);
+  });
+
+
+  it('lists the keys in their natural sort order', function() {
+    eq(R.keys({a: 1, d: 2, c: 3, e: 4, b: 5}), ['a', 'b', 'c', 'd', 'e']);
+  });
+
+  it('places numeric indices first, but in lexicographic order', function() {
```

Comment:
Actually, I'd rather use this, and include it in the example, dropping "numeric", but keeping "lexicographic" (or "Unicode", or some other functionally equivalent wording).  I think it's useful to note.

This also reminds me that I didn't update any docstrings.  Obviously at least one more commit to come.